index.html

```diff
@@ -61,34 +61,47 @@ <h1 class="display-3">Bikes for Refugees</h1>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<div class="row justify-content-between">
+						<div class="col-md-3 col-sm-1 check">Check availavity</div>
+						<div class="col-md-3 col-sm-1 donate">Donate bikes</div>
+						<div class="col-md-3 col-sm-1 volunteer">Volunteer with us</div>
+					</div>
 					<h2 class="heading-underline">Learn more</h2>
 					<div class="articles">
-						<article class="article">
-							<div class="article-title">
-								Why do refugees need bikes?
-							</div>
-							<p class="article-lead">Many refugees are placed in housing in areas where there are few jobs. Bikes provide low-cost transportation so that they can get to work, manage child care and do their shopping.</p>
-							<p class="article-read-more">
-								<a href="#">
-									<span class="fa fa-caret-right"></span>
-									Learn more
-								</a>
-							</p>
-						</article>
-						<article class="article">
-							<div class="article-title">
-								How can I help?
+						<div class="row justify-content-between">
+							<div class="col-md-5 col-sm-3 column-1">
+								<article class="article">
+										<div class=" article-title">
+											Why do refugees need bikes?
+										</div>
+										<p class="article-lead">Many refugees are placed in housing in areas where there are few jobs. Bikes provide low-cost transportation so that they can get to work, manage child care and do their shopping.</p>
+										<p class="article-read-more">
+											<a href="#">
+												<span class="fa fa-caret-right"></span>
+												Learn more
+											</a>
+										</p>
+									</article>
+								</div>
+								<div class="col-md-5 col-sm-3 column-2">
+										<article class="article">
+												<div class="article-title">
+													How can I help?
+												</div>
+												<p class="article-lead">We need lots of bikes and bike accessories! If you have an old bike you don't use, bring it to one of our dropoff events. Or come attend one of our fundraisers.</p>
+												<p class="article-read-more">
+													<a href="#">
+														<span class="fa fa-caret-right"></span>
+														Learn more
+													</a>
+												</p>
+											</article>
+										</div>
+									</div>
+									
 							</div>
-							<p class="article-lead">We need lots of bikes and bike accessories! If you have an old bike you don't use, bring it to one of our dropoff events. Or come attend one of our fundraisers.</p>
-							<p class="article-read-more">
-								<a href="#">
-									<span class="fa fa-caret-right"></span>
-									Learn more
-								</a>
-							</p>
-						</article>
-					</div>
-				</div>
+						</div>
+						
 				<div class="sidebar-content col-lg">
 					<h2 class="heading-underline">Upcoming events</h2>
 					<div class="articles">
```

Comment:
the 3 items under Upcoming Events are not aligned with the heading.
Hint: Avoid using `row` without `col-` classes inside - it does strange things!